File: Java/Card.java
public class Card extends Payment {
  Integer numberCard;
  Integer cvv;
  String expiration;
  Float amount;

  public Card(Integer numberCard, Integer cvv, String expiration, Float amount) {
    super();
    this.numberCard = numberCard;
    this.cvv = cvv;
    this.expiration = expiration;
    this.amount = amount;
  }

}

File: JavaScript/Card.js
class Card extends Payment {
  constructor(numberCard, cvv, expiration, amount) {
    super();
    this.numberCard = numberCard
    this.cvv = cvv
    this.expiration = expiration
    this.amount = amount
  }
  printCard() {
    console.log({
      numberCard: this.numberCard,
      cvv: this.cvv,
      expiration: this.expiration
    });
  }
}